rdrf/rdrf/templates/rdrf_cdes/navbar.html

```diff
@@ -21,7 +21,7 @@
         {% endif %}
       </a>
       <button type="button" class="navbar-toggler" data-bs-toggle="collapse" data-bs-target="#navbar-collapsing-links" aria-controls="navbar-collapsing-links" aria-expanded="false" aria-label="Toggle navigation">
-          <span class="fa fa-bars"></span>
+          <span class="fa fa-chevron-down"></span>
```

Comment:
What do you think of replacing the navbar hamburger with a chevron like this?

Reply:
I think the chevron looks good & clean. My only concern is that we are deviating from the "universally recognised" icon for a menu button in the mobile context. But in saying that, a chevron isn't too obscure it would probably work just as well.

Reply:
Ned to update the navbar on the landing page for consistency.

Reply:
Yeah, maybe it's worth finding a more distinct icon for the form links instead?

Reply:
Also, one or both of the icons could be accompanied by text labels for clarity.